src/mahoji/commands/runecraft.ts

```diff
@@ -71,9 +77,15 @@ export const runecraftCommand: OSBMahojiCommand = {
 		userID,
 		options,
 		channelID
-	}: CommandRunOptions<{ rune: string; quantity?: number; usestams?: boolean; daeyalt_essence?: boolean }>) => {
+	}: CommandRunOptions<{
+		rune: string;
+		quantity?: number;
+		usestams?: boolean;
+		runners?: boolean;
+		daeyalt_essence?: boolean;
+	}>) => {
 		const user = await mUserFetch(userID.toString());
-		let { rune, quantity, usestams, daeyalt_essence } = options;
+		let { rune, quantity, usestams, runners, daeyalt_essence } = options;
 
```

Comment:
Needs to be an ironman check somewhere and force runners to false if the minion is ironman (also this is a bad pr)